src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -14749,7 +14798,16 @@ Label900: ;
 		// For debugging only
 		//CALL  ShowWarningError('LookupSItoIP in: ' // TRIM(stringInWithSI) // ' out: ' // TRIM(stringOutWithIP))
 		//IF (foundConv .NE. 0) CALL  ShowWarningError('   Hint ' // TRIM(UnitConv(foundConv)%hint) // IntToStr(foundConv) )
+
 		unitConvIndex = selectedConv;
+
+		// Add warning if units not found. Exclude stirngs without square brackets, or Rating metrics that are inheirently IP (e.g. SEER, IEER, EER, IPLV)
+		if (unitConvIndex == 0 && ! noBrackets && unitSIOnly != "BTU/W-H") {
+			ShowWarningError("Unable to find a unit conversion from " + stringInWithSI + " into IP units");
+			ShowContinueError("Applying default conversion factor of 1.0");
+			ShowContinueError("Please report to https://github.com/NREL/EnergyPlus/issues");
```

Comment:
@nealkruis @JasonGlazer I would drop the last two lines of the warning.  Can't find a conversion, so it won't be converted.  I don't think it adds anything for the user to say more.